Formula/l/llvm.rb

```diff
@@ -449,16 +446,11 @@ def install
       xctoolchain.parent.install_symlink xctoolchain.basename.to_s => "LLVM#{soversion}.xctoolchain"
 
       # Write config files for each macOS major version so that this works across OS upgrades.
-      # TODO: replace this with a call to `MacOSVersion.kernel_major_version` once this is in a release tag:
-      #   https://github.com/Homebrew/brew/pull/18674
-      {
-        11 => 20,
-        12 => 21,
-        13 => 22,
-        14 => 23,
-        15 => 24,
-      }.each do |macos_version, kernel_version|
-        write_config_files(macos_version, kernel_version, Hardware::CPU.arch)
+      MacOSVersion::SYMBOLS.each_key do |v|
+        macos_version = MacOSVersion.new(v)
+        next if macos_version < "10.14" # system headers are in `/usr` before this.
```

Comment:
Thinking about this more, < 10.14 will just fallback to unversioned triple so we should probably move this conditional into `write_config_files` and write `--sysroot=/` `if macos_version < "10.14"`

Reply:
Thanks!